docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -98,7 +98,9 @@ Preparing For a Release
       git clone git@github.com:ClusterHQ/flocker.git "flocker-${VERSION}"
       cd flocker-${VERSION}
       vagrant up
-      vagrant ssh -c "echo export VERSION=${VERSION} >> .bashrc"
+      vagrant scp default:/home/vagrant/.bashrc vagrant_bashrc
```

Comment:
The prerequisites section above already tells the release engineer to install the plugin.